exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<p>I think what is below is the solution to the Exercises, right?  I think this would go well as a companion notebook in the PR. E.g., UV-exercise-solutions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>